gapir/cc/grpc_replay_service.h

```diff
@@ -39,8 +37,8 @@ class ReplayResponse;
 namespace gapir {
 
 using ReplayGrpcStream =
-    grpc::ServerReaderWriter<replay_service::ReplayResponse,
-                             replay_service::ReplayRequest>;
+    grpc_impl::ServerReaderWriter<replay_service::ReplayResponse,
```

Comment:
Why is this needed? In the fix description you talk about upgrading the gRPC version but i don't see why you need to call the _impl directly here.

Reply:
Guess this goes for the changed include/declaration above too.

Reply:
This change is needed to be able to compile, the ServerReaderWriter template has been renamed in the new version.

Basically all the changes of this PR are there to be able to pass build with the new grpc version.